### PR TITLE
Show search results even if there's a 403 error on image loading

### PR DESCRIPTION
**Pull request by ben-rabid**
Micky reported that searching for "hot" wasn't showing any results on https://www.nzsl.nz/signs/search?utf8=%E2%9C%93&s=hot&hs=&l=&lg=&tag=&usage=

The issue I have found is the Hot water beach sign causing a 403 error which seems to stop foundation triggering the call to show the results.

I have removed the "show if only one" limit to show the search results regardless of count - this does occasionally mean they look a bit uneven, but it's better than no results.

### CODE REVIEW

File: app/assets/javascripts/foundation-equalizer.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
 $(document).ready(function() {
-    if ($("#search-container").children().length === 1) {
+    if ($("#search-container").children().length) {
```
Comment (KatieWright26) [inline app/assets/javascripts/foundation-equalizer.js]: Remove conditional otherwise equalizer wont run

**Comment by ben-rabid** [comment]
This looks like a bigger PR than it is because I indented the JS - all that's new is the removal of `console.log` and the on `window` block

**Comment by ben-rabid** [comment]
@Br3nda do you know what hound is annoyed about?

**Comment by Br3nda** [comment]
the hound error should go away once #110 is merged

**Pull request merged**